pstress/pstress-run.sh

```diff
@@ -36,6 +36,14 @@ if [[ ${SIGNAL} -ne 15 && ${SIGNAL} -ne 4 && ${SIGNAL} -ne 9 ]]; then
   exit
 fi
 
+# Disable TokuDB in case RocksDB is enabled
+# Disable encryption in case RocksDB is enabled
```

Comment:
why do we need to disable keyring plugin to use RocksDB?

Reply:
The reason is that RocksDB tables does not support encryption. Starting the server with keyring plugin or keyring component won't be useful.